modules/apps/portlet-configuration/portlet-configuration-web/src/main/java/com/liferay/portlet/configuration/web/internal/portlet/PortletConfigurationPortlet.java

```diff
@@ -604,6 +606,15 @@ public void updateRolePermissions(
 			}
 		}
 
+		PortletConfigurationPermissionPropagation
+			portletConfigurationPermissionPropagation =
+				PortletConfigurationPermissionPropagationTracker.
+					getPortletConfigurationPermissionPropagation(
+						portletResource);
+
+		portletConfigurationPermissionPropagation.updatePermissionPropagation(
+			actionRequest, actionResponse);
+
```

Comment:
Should check null to avoid NullPointerException.

Reply:
Fixed it! Thanks